liberty-maven-plugin/pom.xml

```diff
@@ -251,7 +245,6 @@
                                     <pomExclude>server-param-default-it/pom.xml</pomExclude>
                                     <pomExclude>server-param-configdir-not-override-it/pom.xml</pomExclude>
                                     <pomExclude>install-features-it/pom.xml</pomExclude>
-                                    <pomExclude>generate-features-it/pom.xml</pomExclude>
```

Comment:
There is still two more references to `generate-features-it` in this pom.xml. Are they there on purpose?

Reply:
Yes I believe those can be left in, they are to omit the generate-features-it tests from the profiles `windows-ol-its` and `windows-wlp-its` (the same profiles we omit the dev-it tests from). Then, I'm not sure that those profiles are being called anywhere, they are not running through GitHub actions currently.

With the changes in this pull request, the generate-features-it will still run but all of the test classes are skipped.